nilearn/maskers/multi_nifti_masker.py

```diff
@@ -62,6 +61,38 @@ def _get_mask_strategy(strategy):
         )
 
 
+def _make_brain_mask_func(mask_type):
```

Comment:
Could this function have a `multi=True` parameter to return the correct function so that the two functions (the one defined below in nifti_masker) can be unified?

Reply:
will do